File: FoodOrderSystemAPI.BL/DTOs/Restaurants/RestaurantsReadDto.cs
namespace FoodOrderSystemAPI.BL;

public class RestaurantsReadDto
{
    public string RestaurantName { set; get; } = string.Empty;
    public string Address { set; get; } = string.Empty;
    public string Logo { set; get; } = string.Empty;
    public string Phone { set; get; } = string.Empty;
    public string PaymentMethods { set; get; } = string.Empty;
}

File: FoodOrderSystemAPI.FrontEnd/src/app/types/Restaurant/Restaurants-Read-dto.ts
export class RestaurantDetailsDto {
    constructor(
        public restaurantName: string = '',
        public address: string = '',
        public logo: string = '',
        public phone: string = '',
        public paymentMethods: PaymentType = PaymentType.Cash
    ) {}
}

export enum PaymentType {
    Cash,
    Credit,
    Both
}
